default/generated/azure/tests/data/07-azure-message-queue/application-prod.properties

```diff
@@ -0,0 +1,2 @@
+spring.kafka.properties.schema.registry.url=http://localhost:8081
+spring.rabbitmq.password=rabbit-pass
```

Comment:
same as above

Reply:
sure, replaced all existing passwords to placeholders